simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFragmentBookDetail.kt

```diff
@@ -355,13 +355,13 @@ class CatalogFragmentBookDetail : Fragment() {
 
     val context = this.requireContext()
     this.format.text = when (feedEntry.probableFormat) {
-      null,
       BookFormats.BookFormatDefinition.BOOK_FORMAT_EPUB ->
         context.getString(R.string.catalogBookFormatEPUB)
       BookFormats.BookFormatDefinition.BOOK_FORMAT_AUDIO ->
         context.getString(R.string.catalogBookFormatAudioBook)
       BookFormats.BookFormatDefinition.BOOK_FORMAT_PDF ->
         context.getString(R.string.catalogBookFormatPDF)
+      else -> ""
```

Comment:
I don't like this use of `else`. The reason for actually naming all of the cases specifically instead of using a fallback `else` is so that, when we add a new book format in the future, the compiler will highlight this `when` expression as now being non-total.

Reply:
Makes sense, okay to switch it to an explicit `null -> ""`?

Reply:
Good feedback! Changing to explicit `null -> ""`.